Detectors/FIT/FT0/simulation/src/Digitizer.cxx

```diff
@@ -52,16 +52,16 @@ auto Digitizer::get_time(const std::vector<double>& times, double deadTime) -> C
       if (!result.particle) {
         result.particle = time;
       }
-      result.deadTime = time + 15.6;
-      time += 15.6 - 0.005;
-      is_positive = false;
+      result.deadTime = time + parameters.mCFDdeadTime;
+      time += parameters.mCFDdeadTime - 0.005;
+      is_positive = true;
     } else
       is_positive = cfd_val > 0;
   }
   if (!result.particle) {
     LOG(INFO) << "CFD failed to find peak ";
-    for (double t : times)
-      LOG(DEBUG) << t << ", ";
+    // for (double t : times)
+    // LOG(INFO) << t << ", dead time "<<deadTime;
   }
   return result;
 }
```

Comment:
I think the Digitizer::measure_amplitude is a misleading name, because you are summing up the amplitudes from different time bins, so in fact this is charge integrator, not amplitude measurement.